test/integration/targets/dnf/tasks/repo.yml

```diff
@@ -337,3 +337,77 @@
       dnf:
         name: dinginessentail
         state: absent
+
+- block:
+    - name: Install dinginessentail-1.0-2
```

Comment:
test looks good, but i would put the exact versions in variables to make future changes easier as the specific versions become unavailable.

Reply:
> as the specific versions become unavailable

These are the fake packages from fake repo created by us via `test/integration/targets/setup_rpm_repo/library/create_repo.py`.